WARNING  pytest_result_log:plugin.py:122 --------Start: api_tests/test_novele_web.py::test_novele_website_status---------
WARNING  pytest_result_log:plugin.py:128 ---------End: api_tests/test_novele_web.py::test_novele_website_status----------